helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterContext.java

```diff
@@ -57,20 +63,29 @@ public class ClusterContext {
   private final Map<String, Integer> _estimateUtilizationMap;
   // Cluster total capacity. Used to compute score when sorting replicas.
   private final Map<String, Integer> _clusterCapacityMap;
-
+  private final List<String> _preferredScoringKey;
```

Comment:
Here and in other places, I am a bit confused whey we refer to it as `preferedScoringKey` instead of `preferedScoringKeys` if the user is able to configure multiple scoring keys in the ClusterConfig.

Should we consider changing this to plural for clarity?

Reply:
that's a miss, thanks for catching it

Reply:
changed to plural everywhere